test/specs/default-controls.test.js

```diff
@@ -99,272 +99,66 @@ describe('<PagingDots />', () => {
 });
 
 describe('<NextButton />', () => {
-  describe('#nextButtonDisable for cellAlign = `left`', () => {
```

Comment:
With the new logic, I had to re-write all the `nextButtonDisable` tests and I greatly simplified them.  Just checking for happy path basic logic.

Reply:
👍